### DISCUSSION TITLE
Widget doesn't display test cases

### DISCUSSION THREAD

**Discussion by qr40m** [allure-framework/allure2#2436] [TestOps]
Hi! 
I'm using Top test cases widget with custom fields, but it doesn't show test cases. 
As a filter I use cf (custom fields) and launch ids, please check an attach 
<img width="1032" alt="Screenshot 2024-03-21 at 12 14 39" src="https://github.com/allure-framework/allure2/assets/33320191/84606e9b-8896-4ba8-bd7e-537739bd3b7f">
<img width="1416" alt="Screenshot 2024-03-21 at 12 14 56" src="https://github.com/allure-framework/allure2/assets/33320191/3495f7d0-21e0-4522-8ef0-a653be3bf9fb">

**Comment by cheshi-mantu** [comment]
Hi @qr40m 
the validation fields show all the available data, but all the widgets show only window of data for 14 most recent days in all widgets that do not have adjustable time span (weeks, days, months) in the UI.
see for example
- https://docs.qameta.io/allure-testops/briefly/dashboards/#standard-dashboard
- https://docs.qameta.io/allure-testops/briefly/dashboards/#automation-trend-widget

**Reply by qr40m** [reply]
got it, thank you!
